Homework/homework10/task.py

```diff
@@ -0,0 +1,145 @@
+import asyncio
+import json
+import re
+from datetime import datetime
+from typing import Dict, List, Tuple
+
+import aiohttp
+
+from bs4 import BeautifulSoup
+
+import requests
```

Comment:
`aiohttp` and `requests` doing the same thing.    In general, it is not recommended to use them simultaneously.